README.md

```diff
@@ -197,8 +197,57 @@ mappings:
     idpath: pci-0000:00:14.0-usb-0:5
 ```
 
+## Envs
+
+`envs` are the method by which a labelled port can have a generated environment variable that 
+bears the path to its node i.e. /dev/sda1
+
+using the above raspberry pi example
+``` yml
+      - port: 3
+        label: Raspberry Pi UART
+        env: UART
+```
+
+This will generate a string in the form of `RASPBERRY_PI_UART=/dev/foo` which can then be applied 
```

Comment:
You'll need to explain where the RASPBERRY_PI prefix comes from.

Also won't it be RASPBERRY_PI_UART_0 or similar?